pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/SchemasResourceBase.java

```diff
@@ -114,6 +114,10 @@ public CompletableFuture<SchemaVersion> deleteSchemaAsync(boolean authoritative,
     }
 
     public CompletableFuture<SchemaVersion> postSchemaAsync(PostSchemaPayload payload, boolean authoritative) {
+        if (SchemaType.BYTES.name().equals(payload.getType())) {
+            throw new RestException(Response.Status.NOT_ACCEPTABLE,
```

Comment:
Please return failed future to avoid additional` try-catch` when invoke this method.

Reply:
Ah. Good catch!

+1

Reply:
Oh, understand, fixed.